ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -648,6 +664,94 @@ public void shouldDistributeAvoCreateStatementWithColumns() {
         )), any(), any());
   }
 
+  @Test
```

Comment:
Review Note: TODO - in a follow-up PR when I move all of these tests to the right locations (after refactoring #2502) I will also move these tests. I want to keep jitter on this small after getting the green ticks